cmd/attest.go

```diff
@@ -156,6 +174,10 @@ func addKeyFlag(cmd *cobra.Command) {
 	cmd.PersistentFlags().StringVar(&key, "key", "AK", "indicates type of attestation key to use <gceAK|AK>")
 }
 
+func addTeeTechnology(cmd *cobra.Command) {
+	cmd.PersistentFlags().StringVar(&teeTechnology, "tee_technology", "", "indicates the type of TEE hardware. Should be empty or sev-snp")
```

Comment:
Based on the current naming convention for this CLI, it should be "tee-technology"

Reply:
I have changed the flag name to `--tee-technology` to make it consistent throughout the file.